README.md

```diff
@@ -74,26 +135,17 @@ protected void onCreate(Bundle savedInstanceState) {
 
 ##### For standalone components
 
-1. [Provide `rootProject.ext.adyenReactNativeRedirectScheme`](https://developer.android.com/studio/build/manage-manifests#inject_build_variables_into_the_manifest) to your App's manifests.
-   To do so, add following to your **App's build.gradle** `defaultConfig`
-```groovy
-defaultConfig {
-    ...
-    manifestPlaceholders = [redirectScheme: rootProject.ext.adyenReactNativeRedirectScheme]
-}
-```
-
-2. Add `intent-filter` to your Checkout activity:
+1. Add `intent-filter` to your Checkout activity:
 ```xml
 <intent-filter>
     <action android:name="android.intent.action.VIEW" />
     <category android:name="android.intent.category.DEFAULT" />
     <category android:name="android.intent.category.BROWSABLE" />
-    <data android:host="${applicationId}" android:scheme="${redirectScheme}" />
```

Comment:
Is defining the host not necessary anymore?

Reply:
It is up to merchant. Host is needed to filter intents out (e.x. for DropIn Activity), and we no longer filtering anything.